Yoast/yoast-components
# Add optional help text to contentanalysis

## Summary

This PR can be summarized in the following changelog entry:

* Added a help text component.

## Relevant technical choices:

* Uses PropType one of to make it possible to pass the text in an array as well as just a string. In the array you can pass a link for example. This prop type is exported to be used in a component that uses HelpText.
* Uses a paragraph element with a default style as main parent.

## Test instructions

This PR can be tested by following these steps:

* View the HelpText in action: `yarn start` -> `localhost:3333` -> tab `Content analysis`.
* Check `yarn test` && `grunt check`.

Fixes #476
